src/Components/Search/SearchBar.tsx

```diff
@@ -400,19 +400,19 @@ export class SearchBar extends Component<Props, State> {
       value: term,
     }
 
-    const firstSuggestionPlaceholder = {
+    const seeAllResultsPlaceholder = {
       node: {
         displayLabel: term,
-        displayType: "FirstItem",
+        displayType: "SeeFullResultsSuggestionItem",
```

Comment:
**suggestion/blocking**: I think this has the risk of altering some analytics to track when a user has clicked on the "See full results" suggestion. For now, can we keep the displayType as `FirstItem` until we're able to touch base with @abhitip and align on a new value?

Reply:
It makes sense. I am reverting back

Reply:
> reverting back

I think we can rename it in other places, but leave only the old value (`FirstItem`) here

Reply:
> I think this has the risk of altering some analytics to track when a user has clicked on the "See full results" suggestion

Yes, you're right

<img width="1281" alt="Sources - force-staging - Segment 2023-03-15 13-53-23" src="https://user-images.githubusercontent.com/3513494/225288209-55b20248-1e31-44fd-b809-03a46aa18a85.png">

Reply:
Yeah, exactly. The code variables can be renamed/refactored as long as the analytics event remains consistent.